src/EPR.ProducerContentValidation.Application/Validators/GroupedValidators/WarningValidators/SinglePackagingMaterialGroupedValidator.cs

```diff
@@ -48,7 +48,7 @@ public override async Task ValidateAsync(List<ProducerRow> producerRows, string
             }
 
             // Two First() methods called as we need the first item of the first group of data (only one group exists)
-            var representativeRow = distinctPackagingMaterialRows.First().First();
+            var representativeRow = distinctPackagingMaterialRows[0].First();
```

Comment:
could you please write 2 unit test on **old code** i.e.

var representativeRow = distinctPackagingMaterialRows.First().First();
var packagingMaterialSubtypes = distinctPackagingMaterialRows
                .First()
                .GroupBy(row => row.MaterialSubType)
                .ToList();
                
test 1: distinctPackagingMaterialRows.First() is null
Test 2 distinctPackagingMaterialRows.First() is empty
and then do your code change and re-run the tests and see what happens?